bootstrap/create-multi-key-account.go

```diff
@@ -283,3 +382,57 @@ transaction(publicKeys: [Crypto.KeyListEntry], contracts: {String: String}, fund
 	}
 }
 `)
+
+var createAndFundCloudKMS = []byte(`
+import Crypto
+import "FlowToken"
+import "FungibleToken"
+
+transaction(contracts: {String: String}, fundAmount: UFix64) {
+    let tokenReceiver: &{FungibleToken.Receiver}
+    let sentVault: @{FungibleToken.Vault}
+
+	prepare(signer: auth(BorrowValue) &Account) {
+		let account = Account(payer: signer)
+
+		let publicKeys: [String] = [
+            "3549d9d17014d02feb159c5069fd79c2290b075cce8496c476dad6aadad4cefb91928c1ab2709652fc46cfbafb8bac89844a305da3382c4aebb13e9525698daa",
```

Comment:
@sideninja These 5 public keys exist on my personal Google Cloud KMS account, we'll need to generate some others to be able to run this in CI.

Reply:
this should be passed in as an argument to the command

Reply:
Good point :+1: 
Updated in https://github.com/onflow/flow-evm-gateway/pull/282/commits/114dde2dcc765a39bca1276d1038343501340496